x/auth/ante.go

```diff
@@ -95,7 +95,7 @@ func NewAnteHandler(am AccountKeeper, fck FeeCollectionKeeper) sdk.AnteHandler {
 			fck.AddCollectedFees(newCtx, stdTx.Fee.Amount)
 		}
 
-		for i := 0; i < len(stdSigs); i++ {
+		for i := 0; i < len(stdSigs) && i < ctx.LimitTxSigs(); i++ {
```

Comment:
Do we need to handle recursive multisigs? I can't recall.

Also, I think we want to actually return an error if more than the limit is provided. Correct me if I'm wrong @cwgoes.

Reply:
> Also, I think we want to actually return an error if more than the limit is provided. Correct me if I'm wrong

Unsure about that, I'll wait 'til @cwgoes shed light on that